zypper-docker.go

```diff
@@ -37,3 +44,21 @@ func main() {
 	app := newApp()
 	app.RunAndExitOnError()
 }
+
+func ListenSignals() {
```

Comment:
- Don't export it.
- Put it either in the `client.go` file or in a new file `signals.go`.